### PR TITLE
London Class 7 - Rafael Flores - GIT/GitHub - Week 1

### PR DESCRIPTION
**Pull request by rfloresram**
(Linus is not the most smart kid of the class)

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -4,110 +4,112 @@
...
-Making a website go faster
+Making a website looks pretty, the hype for graphic designers and editors.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
+<h1> { Here is the section for heading elements
```
Comment (iTrip23) [inline homework.md:38]: Hi Rafael,
nice touch with the answers so far :),
I think you are using html tags in css, not sure if that is correct, should be: `h1 {
//css code here
}`
what do you think?

File: homework.md [hunk 2]
```diff
@@ -4,110 +4,112 @@
...
 
-A place for us to store and share our code.
+GitHub is a code hosting platform for version control and collaboration.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Means send your changes for evaluation of the crew or feedback, from your local computer to GitHub on the web as a remote repo.
```
Comment (iTrip23) [inline homework.md:97]: Hi Rafael,
check the difference between commit and pull requests, should help.
great job so far

**Comment by iTrip23** [comment]
Hi Rafael,
Nice job with the answers, you had some creative ones in there :).
Have another look at the answers for questions 6 and 15, I could be wrong...

File: homework.md (hunk 2 above)
Comment (rfloresram) [reply homework.md:97]: Indeed, I will make the changes on it. Thanks bud! 💯

File: homework.md (hunk 1 above)
Comment (rfloresram) [reply homework.md:38]: Thank you, I felt really stocked on it. Great feedback!

File: homework.md [hunk 3]
```diff
@@ -4,110 +4,112 @@
...
 
-Hyper Tall Massive Language
+Hypertext Markup Language (HTML) (I didn't even knew this)
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website looks pretty, the hype for graphic designers and editors.
```
Comment (lpalma) [inline homework.md:31]: You can definetely make your website look pretty with CSS!
There is also another important aspect of it, which explains why we have some people specialized in graphic / web design and some specialized in code / content creation.
Can you tell us what that important aspect is? :)

File: homework.md [hunk 4]
```diff
@@ -4,110 +4,112 @@
...
-}
+} </h1>
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+Javascript is a language that gives interface to the web pages adding interactive elements that catch the attention of the user.
```
Comment (lpalma) [inline homework.md:47]: I like how you describe it using your own words :)
Here's a question for you: can we use JS for creating things other than web pages? ;)

File: homework.md [hunk 5]
```diff
@@ -4,110 +4,112 @@
...
+Javascript is a language that gives interface to the web pages adding interactive elements that catch the attention of the user. 
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Github is a community of developers where ideas and works can be shared.
+Makes easy the collaborators between career colleagues.
+Team members can work on files and they can easily merge changes.
```
Comment (lpalma) [inline homework.md:55]: Cool :clap: 
Git was created to solve some important issues that developers faced in the past. This was covered in the first lesson of the Git week. Can we remember that those things are? Tip: you can always revisit the material ;)
https://syllabus.codeyourfuture.io/git/index

File: homework.md [hunk 6]
```diff
@@ -4,110 +4,112 @@
...
 
-When you send a file over the internet
+You're proposing changes on the document as well asking for a feedback about your collaboration. 
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+A list of the commits you made on the repo; from the most actual to the last made.
```
Comment (lpalma) [inline homework.md:79]: Spot on!
And what is a **commit**? :P

File: homework.md [hunk 7]
```diff
@@ -4,110 +4,112 @@
...
 
-To record what changes you made to the code
+Indicates the people about the work you did in each part.
 
 ## Question 18
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+Idk, you tell me! Im new on these, I will use Google for now and Slack for my mentors and class colleagues :)
```
Comment (lpalma) [inline homework.md:115]: Google and Slack are good places to look for answers.
However, for most of the work we'll do with Git / GitHub, we'll use commands which have been explained in the Git Cheatsheet too. Can you point out where this cheatsheet is? :)

**Pull request closed without merging**